CHANGELOG.md

```diff
@@ -34,6 +34,7 @@ For older changes see the [archived Singularity change log](https://github.com/a
   `index.docker.io`.
 - Skip attempting to bind inaccessible mount points when handling the
   `mount hostfs = yes` configuration option.
+- Support parsing nested variables defined inside definition file.
```

Comment:
This is specifically for `%arguments` section, right?  If so I think it would be good to mention that.

Was this fixing something that was already documented to work, or is it extending the feature a bit?

Reply:
a little follow up:
at least v1.2.5 supported nested (at least 1 nest) variables but user reported v1.3.4 doesn't work as expected.
nested more than 1 nest is feature enhancements since we didn't support that before.

Reply:
thanks for @panda1100 's explanation. Yeah, it's for `%arguments` section, and it'll allow `%arguments` section to define a variable with reference of another variable, but not nested. Remembered that this feature is cherry picked back from Singularity's re-implementation. After the re-impl, the behavior is changed.